vividus-plugin-web-app/src/test/java/org/vividus/bdd/steps/ui/web/PageStepsTests.java

```diff
@@ -330,8 +336,13 @@ void testRefreshPage()
     @Test
     void testOpenPageUrlInNewWindow()
     {
+        InOrder inOrder = Mockito.inOrder(setContextSteps, navigateActions, javascriptActions, uiContext);
```

Comment:
`inOrder` -> `ordered` (PMD doesn't like variables prefixed with `in` and `out`)